include/RAJA/util/Operators.hpp

```diff
@@ -386,7 +386,7 @@ struct logical_and : public detail::comparison_function<Arg1, Arg2>,
   {
     return lhs && rhs;
   }
-  static constexpr const bool identity = true;
+  static constexpr bool identity() {return true;}
```

Comment:
Yeah, got it. I shouldn't try to review PRs while I'm answering emails and talking on the phone at the same time.